concordium-android-sdk/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.concordium.sdk"
+          android:versionCode="1"
+          android:versionName="1.0"
+>
+    <uses-sdk android:minSdkVersion="4" android:targetSdkVersion="16" />
```

Comment:
The versions here should match what we are building for, right?

Reply:
I have changed it to 27.